package NPersonalization.NProto;

import "kernel/user_history/proto/embedding_types.proto";

import "kernel/embeddings_info/proto/embedding.proto";

option go_package = "a.yandex-team.ru/kernel/user_history/proto;userhistory";

message TFadingEmbedding {
    required NEmbedding.TEmbedding Embedding = 1;
    required uint64 LastUpdTimestamp = 2;
    required float FadingCoefDays = 3;
    optional TEmbeddingOptions Options = 8;
}
